src/less/mixins/utility/utility-mixins.less

```diff
@@ -25,8 +25,10 @@
 
 .imageTreatment() {
     align-items: center;
+    border-radius: 8px;
```

Comment:
Can we pass this in as a variable instead?
```
.imageTreatment(@border-radius: 8px) {
  border-radius: @border-radius;
}
```
Something like this, default to 8px and you can override.

Reply:
done - let me know if I need any other changes